api/handlers_e2e_test.go

```diff
@@ -120,7 +120,7 @@ func TestApplicationHandlers(t *testing.T) {
 	block, err := test.MakeBlockForTxns(test.MakeGenesisBlock().BlockHeader, &txn, &optInTxnA, &optInTxnB)
 	require.NoError(t, err)
 
-	err = proc.Process(&rpcs.EncodedBlockCert{Block: block})
+	err = proc.ProcessBlockCert(&rpcs.EncodedBlockCert{Block: block})
```

Comment:
Why not create the BlockData object instead and stick with the generic interface?

Reply:
This was a simple rename of the original interface which had the same name as "Process"